pages/ox_inventory/Functions/Client.mdx

```diff
@@ -443,6 +443,16 @@ exports.ox_inventory:GetSlotsWithItem(itemName, metadata, strict)
 Return:
 - slotsData: `table[]?`
 
+## supressItemNotifications
+
+Allows you to enable or disable NUI item notifications
+
+- value: `boolean`
```

Comment:
Move this below the export codeblock to keep it consistent with other definitions.

Reply:
Fixex